src/test/java/uk/gov/justice/digital/hmpps/cmd/api/domain/DryRunNotificationDescriptionTest.kt

```diff
@@ -0,0 +1,328 @@
+package uk.gov.justice.digital.hmpps.cmd.api.domain
```

Comment:
copy of existing test class but for DryRunNotification rather than the old Notification jpa class